src/UKHO.ERPFacade.Common/IO/ObjectXMLSerializer.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using System.Text;
+using System.Xml.Serialization;
+using System.Xml;
+
+namespace UKHO.ERPFacade.Common.IO
+{
+    [ExcludeFromCodeCoverage]
+    public class ObjectXMLSerializer<T> where T : class
+    {
+        public static string SerializeObject(T serializableObject)
```

Comment:
Need a unit test for this. Can T be null?

Reply:
Unit tests added.